### ISSUE TITLE
ADD FUEL NETWORK TO HOP PROTOCOL

### ISSUE THREAD

**Issue by Femoralnelson** [FuelLabs/sway#5644]
Name of the project
HOP PROTOCOL

Project type
Library

Site of the project
https://hop.exchange/

Github repository of the project
https://github.com/hop-protocol

Networks
fuel Mainnet

Additional context
Hop Protocol is a scalable rollup-to-rollup general token bridge. It enables users to swiftly transfer tokens from one rollup or sidechain to another without waiting for the network’s challenge period. In other words, it allows seamless token movement across different Ethereum scaling solutions. It works by involving market makers (referred to as Bonder) who front the liquidity at the destination chain in exchange for a small fee.
This credit is extended by the Bonder in form of hTokens which are then swapped for their native token counterpart in an AMM.
The end result allows users to seamlessly transfer tokens from one network to the next.
Additionally, Hop Protocol connects Ethereum with Layer-2 networks, reducing the typical wait time for cross-network transactions from multiple days to nearly immediate transfers.
Hop Bridge: A multichain bridge that facilitates token transfers between Ethereum and Layer-2 networks.
Hop Exchange: A platform that allows users to send tokens across Ethereum scaling solutions.

In conclusion, adding Hop Protocol which has a very fast bridging function Would Connect A Long List Of L1 & Non Evm Blockchains To FUEL thereby allowing easy access to FUEL from different blockchains and protocols and also allows a Very Fast Onboarding process for FUEL new users and investors.
